utils/paramfetch/src/lib.rs

```diff
@@ -117,18 +127,21 @@ pub async fn get_params(
 /// Get proofs parameters and all verification keys for a given sector size using default manifest.
 #[inline]
 pub async fn get_params_default(
+    data_dir: &Path,
     storage_size: SectorSizeOpt,
     is_verbose: bool,
 ) -> Result<(), anyhow::Error> {
-    get_params(DEFAULT_PARAMETERS, storage_size, is_verbose).await
+    get_params(data_dir, DEFAULT_PARAMETERS, storage_size, is_verbose).await
 }
 
 async fn fetch_verify_params(
+    data_dir: &Path,
     name: &str,
     info: Arc<ParameterData>,
     mb: Option<Arc<MultiBar<Stdout>>>,
 ) -> Result<(), anyhow::Error> {
-    let path: PathBuf = [&param_dir(), name].iter().collect();
+    let mut path: PathBuf = param_dir(data_dir);
+    path.push(name);
     let path: Arc<Path> = Arc::from(path.as_path());
```

Comment:
Could you refactor this to avoid a mutable variable?

Reply:
Refactored